src/package-linker.js

```diff
@@ -296,7 +296,17 @@ export default class PackageLinker {
     // remove all extraneous files that weren't in the tree
     for (const loc of possibleExtraneous) {
       this.reporter.verbose(this.reporter.lang('verboseFileRemoveExtraneous', loc));
-      await fs.unlink(loc);
+
```

Comment:
`files` set is used just in 4 places.
So yeah, I think doing `files.add(loc.toLowerCase());` and `file.has(loc.toLowerCase())` should work